core-blocks/test/fixtures/core__text__converts-to-paragraph.serialized.html

```diff
@@ -1,3 +1,3 @@
 <!-- wp:paragraph -->
-<p>This is an old-style text block. Changed to <code>paragraph</code> in #2135.</p>
+<p>This is an old-style text block.  Changed to <code>paragraph</code> in #2135.</p>
```

Comment:
Interesting. So beautify is also dropping multiple spaces?

Reply:
Yes, presumably because in a browser there would be no rendered difference (whitespace collapsing).